view-samples/prompt-cards/README.md

```diff
@@ -0,0 +1,85 @@
+# Prompt Cards
+
+## Summary
+This sample demonstrates how to format list items as prompt cards. The design is inspired from the [Copilot Lab](https://copilot.cloud.microsoft/en-US/prompts/all) prompt library. It features the following:
+- **Responsive tile layout:** Items are displayed in a card-based layout that adjusts to different screen sizes.
+- **Conditional Icons:** Icons change dynamically based on the value of the **Prompt Category** column.
+- **Theme Color Classes:** Uses theme color classes to ensure the format displays as intended regardless of the site's theme (light, dark, custom, etc.).
+- **Gallery View:** The formatting is designed to work with the Gallery view style.
+
+![Screenshot of the prompt cards list view formatting.](./assets/screenshot.png)
+
+
+## View requirements
+
+**Important:** This formatting depends on the Gallery view style. Make sure to set your view to Gallery before applying the formatting.
+
+|Type|Internal Name|Required|
+|---|---|:---:|
+|Single line of text|Title|Yes|
+|Multiple lines of text|Prompt|Yes|
+|Choice|Prompt Category|Yes|
+|Choice (allow multiple)|Works in|Yes|
+|Multiple lines of text|Make it your own|No|
```

Comment:
Looking at the JSON, this column did not appear to be used. If you are not using it, please remove this column from the View Requirements. If you are using it, please update the JSON to use it.

Reply:
👍